### PR TITLE
OPSEXP-2664: update APS app probes endpoints

### PR DESCRIPTION
**Pull request by alxgomz**
Ref: OPSEXP-2664

(APS admin dev images are not released so we'll have to wait for release before we can do/test it OPSEXP-2702)

### CODE REVIEW

File: charts/alfresco-activiti/values.yaml [hunk 1]
```diff
@@ -61,12 +61,16 @@ persistence:
...
+  path: /activiti-app/actuator/health/readiness
   initialDelaySeconds: 25
   periodSeconds: 10
   failureThreshold: 5
   timeoutSeconds: 5
-livenessProbe: *apsProbe
+livenessProbe:
+  <<: *apsProbe
+  # -- Make sure to set `readinessProbe.path` to `/activiti-app/app/rest/locale` if you're using a version of APS prior to 24.3.0
+  path: /activiti-app/actuator/health/liveness
```
Comment (pmacius) [inline charts/alfresco-activiti/values.yaml:73]: Shouldn't we have some different initialDelayseconds etc. now when the endpoints are different

Comment (alxgomz) [reply charts/alfresco-activiti/values.yaml:73]: I see no good settings I could change here. even the initial delay is not worth changing given the app runs in tomcat and tomcat would only treat requests after full startup of the app.

**Review by pmacius — APPROVED** [review]

File: charts/alfresco-activiti/README.md [hunk 2]
```diff
@@ -64,11 +64,12 @@ A Helm chart for Alfresco Activiti
...
-| livenessProbe.failureThreshold | int | `5` |  |
-| livenessProbe.initialDelaySeconds | int | `25` |  |
-| livenessProbe.path | string | `"/activiti-app/app/rest/locale"` |  |
-| livenessProbe.periodSeconds | int | `10` |  |
-| livenessProbe.timeoutSeconds | int | `5` |  |
+| livenessProbe.<<.failureThreshold | int | `5` |  |
+| livenessProbe.<<.initialDelaySeconds | int | `25` |  |
+| livenessProbe.<<.path | string | `"/activiti-app/actuator/health/readiness"` | Make sure to set `readinessProbe.path` to `/activiti-app/app/rest/locale` if you're using a version of APS prior to 24.3.0 |
+| livenessProbe.<<.periodSeconds | int | `10` |  |
+| livenessProbe.<<.timeoutSeconds | int | `5` |  |
```
Comment (gionn) [inline charts/alfresco-activiti/README.md:71]: helm-docs doesn't seems to like that syntax, I prefer we just repeat those few keys

Comment (alxgomz) [reply charts/alfresco-activiti/README.md:71]: mmmm Indeed, I missed that

File: charts/alfresco-activiti/values.yaml [hunk 3]
```diff
@@ -60,13 +60,20 @@ persistence:
...
-  path: /activiti-app/app/rest/locale
+readinessProbe:
+  # -- Make sure to set `readinessProbe.path` to `/activiti-app/app/rest/locale` if you're using a version of APS prior to 24.3.0
+  path: /activiti-app/actuator/health/readiness
+  initialDelaySeconds: 25
+  periodSeconds: 10
+  failureThreshold: 5
+  timeoutSeconds: 5
+livenessProbe:
+  # -- Make sure to set `lvenessProbe.path` to `/activiti-app/app/rest/locale` if you're using a version of APS prior to 24.3.0
```
Comment (gionn) [inline charts/alfresco-activiti/values.yaml:71]:
```suggestion
  # -- For APS prior to 24.3.0, path must be set to `/activiti-app/app/rest/locale`
```

File: charts/alfresco-activiti/values.yaml [hunk 4]
```diff
@@ -60,13 +60,20 @@ persistence:
   data:
     mountPath: "/usr/local/data"
     subPath: "alfresco-process-services/process-data"
-readinessProbe: &apsProbe
-  path: /activiti-app/app/rest/locale
+readinessProbe:
+  # -- Make sure to set `readinessProbe.path` to `/activiti-app/app/rest/locale` if you're using a version of APS prior to 24.3.0
```
Comment (gionn) [inline charts/alfresco-activiti/values.yaml:64]:
```suggestion
  # -- For APS prior to 24.3.0, path must be set to `/activiti-app/app/rest/locale`
```

**Pull request merged**